src/CoreBundle/private/modules/CMSModuleChooser/views/readonly.view.php

```diff
@@ -1,29 +1,41 @@
 <?php
+
+/**
+ * @var $oModule \TdbCmsTplModule
+ * @var $oModuleInstance \TdbCmsTplModuleInstance
+ */
+
 $menuPrefix = TGlobal::OutHTML($data['sModuleSpotName']);
 ?>
 <div style="margin-top: 4px;position:relative;z-index:1000">
     <div style="border-color:#<?=$oModuleInstanceColorState; ?>" class="CMSModuleChooserReadOnly"
          spotname="<?=$menuPrefix; ?>">
-        <?php if (!is_null($data['oModule'])) {
-    ?>
+        <?php
+        if (null !== $oModule) {
+            $iconFontCssClass = $oModule->fieldIconFontCssClass;
+            if ('' === $iconFontCssClass) {
+                $iconFontCssClass = '';
```

Comment:
I think this is a missing fallback.

Reply:
Whoops. Forgot to select the default icon. Done.